src/schema/nodes/contributor.ts

```diff
@@ -13,21 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Affiliation, BibliographicName } from '@manuscripts/json-schema'
+import { BibliographicName } from '@manuscripts/json-schema'
 import { NodeSpec } from 'prosemirror-model'
 
 import { ManuscriptNode } from '../types'
 
 interface Attrs {
   id: string
   role: string
-  affiliations: Affiliation[]
+  affiliations: string[]
```

Comment:
are we sure about that type, in the [decode](https://github.com/Atypon-OpenSource/manuscripts-transform/blob/a89e255cd7e3ab0144a0aaadedcc86b148f14965/src/transformer/decode.ts#L840) we pass object array!

Reply:
I'm going to check. During runtime I got arrays of ids. So I added these fixes.

Reply:
Checked. It's array of string there. 
<img width="814" alt="image" src="https://github.com/Atypon-OpenSource/manuscripts-transform/assets/40764175/c6ad2be7-ecb5-4239-a558-3fab7bb0338f">